src/aiod/default/authorization_test.py

```diff
@@ -1,18 +1,34 @@
+import http.client
+from collections import NamedTuple
+from typing import Sequence
```

Comment:
`NamedTuple` to be used for inheritance in this case should come from `typing` instead `collections`. 
```suggestion
from typing import Sequence, NamedTuple
```

Reply:
Yep, looks like my IDE imported the wrong version.